### PR TITLE
Adjust tutorial to be similar to the reference one

### PR DESCRIPTION
**Pull request by mosampaio**
I used the php lavarel as reference tutorial to adjust this one.

### CODE REVIEW

File: AccountVerification.Web/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-﻿<h1>Account Verification with Ruby on Rails</h1>
+﻿<h1>Account Verification with C# / ASP.NET MVC</h1>
```
Comment (acamino) [inline AccountVerification.Web/Views/Home/Index.cshtml:1]: Thanks! 👍

This change should be applied also in the `master` branch.

File: tutorial/index.jade [hunk 2]
```diff
@@ -5,99 +5,56 @@
...
-      1. It communicates with Authy's API to check that the given code is correct
-
-      1. Once the code is verified with Authy we send a confirmation SMS to the user.
-
-      1. Finally it redirects the user to a success page.
-
-  .step(data-title='Twilio and Authy configuration' data-file='AccountVerification.Web/Web.config' data-highlight="19-22" data-mode='xml')
-    :markdown
-      ## Twilio and Authy configuration
+      ## Authy configuration
```
Comment (acamino) [inline tutorial/index.jade]: @mosampaio could you please use the proper casing in this title?

Comment (acamino) [inline tutorial/index.jade]: In this step, in the line 62 you can specify `xml` as the language for this code snippet.  Something like:

``````
```xml
   ... Your snippet
``````

File: tutorial/index.jade [hunk 3]
```diff
@@ -112,67 +69,113 @@
...
+      controller. When the code is sent we redirect to another page where the
+      user can enter the token they received, therefore completing the verification process.
 
-      Finally, we call the `SendSms` with the `AuthyClient` which requires the user's
-      `AuthyUserId` and a `true` value for the `force` parameter. This will send the verification code to the user and force it to be sent via SMS.
+      Let's check out the code for that next!
 
-  .step(data-title='Verification code confirmation' data-file='AccountVerification.Web/App_Start/IdentityConfig.cs'
-        data-highlight="113-129")
+  .step(data-title='Verifying the Code', data-file='AccountVerification.Web/App_Start/IdentityConfig.cs', data-highlight='71-89')
```
Comment (acamino) [inline tutorial/index.jade]: Could you please take a look at this one? Seems like the step and the highlighted code are unrelated.

File: tutorial/index.jade [hunk 4]
```diff
@@ -112,67 +69,113 @@
...
+      uses the same Authy API method we used earlier to resend the code.
 
-      **See Also**
-      * [Sending SMS & MMS with Twilio C#](https://www.twilio.com/docs/api/rest/sending-messages#example-1)
+      To wrap things up, let's implement the last step where we confirm that the
+      user's account has been verified with a text message and a success page.
+
+  .step(data-title="Show User Details Page", data-file='AccountVerification.Web/Views/Account/Status.cshtml')
+    :markdown
+      ## Show User Details Page
```
Comment (acamino) [inline tutorial/index.jade]: Do you think this step is relevant in the tutorial?

Comment (mosampaio) [inline tutorial/index.jade]: 25% relevant, but the reference tutorial has this step.

File: AccountVerification.Web/Views/Home/Index.cshtml (hunk 1 above)
Comment (mosampaio) [inline AccountVerification.Web/Views/Home/Index.cshtml:1]: I just created another pull request for it.

**Comment by acamino** [comment]
Thanks @mosampaio!

**Pull request merged**